.idea/workspace.xml

```diff
@@ -1,17 +1,31 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="ChangeListManager">
-    <list default="true" id="9e5680cd-8407-4143-bec1-26afffb8c9b5" name="Changes" comment="wer">
+    <list default="true" id="9e5680cd-8407-4143-bec1-26afffb8c9b5" name="Changes" comment="1. Added spaces inside of the name exported.&#10;2. Moved function to the same line describe.">
       <change beforePath="$PROJECT_DIR$/.idea/workspace.xml" beforeDir="false" afterPath="$PROJECT_DIR$/.idea/workspace.xml" afterDir="false" />
```

Comment:
Don't push this file

Reply:
Resolved in https://github.com/VladyslavSamkov/vladyslav_samkov_cypress/pull/6/files